csrc/options.h

```diff
@@ -21,6 +21,7 @@ namespace nvfuser {
 //! These can be set through the `NVFUSER_DUMP` environment variable
 //!
 enum class DebugDumpOption {
+  CallStack, //!< Dump the call stack of selected internal function
```

Comment:
I'd make some tests. In fact, I just noticed another flag `NVFUSER_DUMP=perf_debug_verbose` has been broken for a long time due to lack of testing :( Testing should be relatively easy with OptionGuard and `captureStdout`.

Reply:
Added `TEST_F(NVFuserTest, FunctionTrace)`.